src/App.js

```diff
@@ -27,6 +29,17 @@ function App() {
         <Route path="/" exact component={LandingPage} />
         <Route path="/metric" exact component={MetricPage} />
         <Route path="/dietaryrestrictions" exact component={DietaryRestrictions} />
+        <Route 
+          path="/mainrecipe" 
+          render={(props) => (
+            <MainRecipeModal 
+              name="Creamy Watermelon Smoothie" 
+              percentage="87%"
+              price="$"
+              img={smoothie} 
+            />
```

Comment:
later when we implement API and data handling, the structure of this will have to change. But it is ok for now 👍